packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -32,6 +32,11 @@ double _getCssOpacity(Color color) {
   return color.opacity;
 }
 
+// Converts a [Color] into a valid CSS value rgba(R, G, B, A).
+String _getCssColorWithAlpha(Color color) {
+  return 'rgba(${color.red}, ${color.green}, ${color.blue}, ${color.alpha / 255})';
```

Comment:
Depending on the alpha, `color.alpha / 255` may end up with a bunch of not very meaningful decimal digits; I normally do `(color.alpha / 255).toStringAsFixed(2)` to limit this.